pkgs/tools/misc/haste-client/Gemfile

```diff
@@ -1,2 +1,2 @@
 source 'https://rubygems.org'
```

Comment:
Why did the Gemfile and Gemfile.lock files change permissions to 755 (from 644)?